test/test.py

```diff
@@ -264,7 +315,7 @@ def _assert_bundle(self, uuid, version, manifest, metadata_files,
         self.assertEqual(storage_methods, {s.storage_method for s in bundle.specimens})
         self.assertEqual(preservation_methods, {s.preservation_method for s in bundle.specimens})
 
-        print(json.dumps(as_json(bundle), indent=4))
+        # print(json.dumps(as_json(bundle), indent=4))
```

Comment:
This does actually have a purpose but it wasn't clear.

```suggestion
        # Prove that as_json returns a valid JSON structure (no cycles, correct types, etc.)
        self.assertTrue(isinstance(json.dumps(as_json(bundle), str))
```

This change is unrelated, so it should go into a separate commit.